test/OpenTelemetry.Instrumentation.W3cTraceContext.Tests/OpenTelemetry.Instrumentation.W3cTraceContext.Tests.csproj

```diff
@@ -36,13 +36,6 @@
     <ProjectReference Include="$(RepoRoot)\test\TestApp.AspNetCore.3.1\TestApp.AspNetCore.3.1.csproj" />
   </ItemGroup>
 
-  <ItemGroup Condition="'$(TargetFramework)' == 'netcoreapp2.1'">
-    <PackageReference Include="Microsoft.AspNetCore.App" Version="[2.1.1,2.2)" >
-      <AllowExplicitVersion>true</AllowExplicitVersion>
-    </PackageReference>
-    <ProjectReference Include="$(RepoRoot)\test\TestApp.AspNetCore.2.1\TestApp.AspNetCore.2.1.csproj" />
```

Comment:
Can the TestApp.AspNetCore.2.1 project be removed with this PR?

Reply:
yes. ! pushing them now!